android/app/src/main/java/org/openbot/env/IVideoServer.java

```diff
@@ -5,10 +5,10 @@
 import android.view.TextureView;
 
 public interface IVideoServer {
-  void startServer();
```

Comment:
Why is this no longer needed?

Reply:
I see, this seems to be handled now in RTSP server with `private void startServer` and is not part of the interface anymore.

Reply:
> Why is this no longer needed?

The server is started implicitly now when all the input conditions are met. Take a look at how `AndGate` is used.

Reply:
On second thought, maybe we should keep this in the interface. Looks like a method you would normally use. I think I will change it back.